RecoLocalCalo/HcalRecAlgos/interface/HcalPipelinePedestalAndGain.h

```diff
@@ -0,0 +1,41 @@
+#ifndef RecoLocalCalo_HcalRecAlgos_HcalPipelinePedestalAndGain_h_
+#define RecoLocalCalo_HcalRecAlgos_HcalPipelinePedestalAndGain_h_
+
+// HB/HE channel information stored for each pipeline "capacitor id"
+class HcalPipelinePedestalAndGain {
```

Comment:
This seems to always (often, at least) be used as an `std::array<HcalPipelinePedestalAndGain, 4>`.
You could add at the end of this header file something like
```c++
constexpr int HcalPipelinePedestalAndGainSize = 4;
using HcalPipelinePedestalAndGainArray = std::array<HcalPipelinePedestalAndGain, 4>;
```
and use those in the rest of the code ?